x-pack/plugins/data_enhanced/public/search/sessions_mgmt/lib/api.ts

```diff
@@ -151,6 +155,7 @@ export class SearchSessionsMgmtAPI {
   }
 
   public reloadSearchSession(reloadUrl: string) {
```

Comment:
Haven't we removed "reload" action and this method and all that related to reload tracking is redundant?

Reply:
You can still reload from management when navigating to an expired search session

Reply:
Aha,

so  `this.deps.usageCollector?.trackSessionReloaded()` is also used in `x-pack/plugins/data_enhanced/public/search/sessions_mgmt/lib/get_columns.tsx` (clicking on a experied session name) - got it 👍 

that is just just a matter of removing this method here `reloadSearchSession` and it is not related to this pr at all.